### PR TITLE
Improve `ConFIG` implementation

### PR DESCRIPTION
**Pull request by PierreQuinton**
* Improve implementation of the matrix-vector product to simply use a single @.
* Change the vector normalization to use torch.nn.functional.normalize with the default epsilon of 1e-12. This should change the output of the aggregator for very uncertain vectors to be zero instead.
* Add changelog entry

### CODE REVIEW

**Review by ValerianRey — APPROVED** [review]
GPU tests all pass.
